src/main/webapp/app/shared/statistics-graph/statistics-average-score-graph.component.ts

```diff
@@ -84,7 +84,10 @@ export class StatisticsAverageScoreGraphComponent implements OnInit {
         private navigationUtilService: ArtemisNavigationUtilService,
         private exerciseTypeFilter: ChartExerciseTypeFilter,
         private categoryFilter: ChartCategoryFilter,
```

Comment:
```suggestion
        readonly exerciseTypeFilter: ChartExerciseTypeFilter,
        readonly categoryFilter: ChartCategoryFilter,
```

Wouldn't this be possible, too? Then we could remove the redundant definition.

Reply:
You're right! I've changed it.